.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/
 composer.lock
+/.idea/
```

Comment:
this is personnal ide config specific, please use your global [gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).